"use strict";
//? FunciÃ³n anonÃ­ma auto invocada
(() => {
    const displayError = (message) => {
        throw new Error(message);
    };
    displayError('Auxilio ðŸ¥¹');
})();
